apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -115,16 +115,23 @@ function DescriptionSection(props: DescriptionProps) {
   );
 }
 
-type CtaProps = Pick<Twc2SupportDetail, 'ctaBlurb' | 'ctaButtonText'>;
+type CtaProps = Pick<
+  Twc2SupportDetail,
+  'ctaBlurb' | 'ctaButtonText' | 'ctaLink'
+>;
 
 function CtaSection(props: CtaProps) {
-  const { ctaBlurb, ctaButtonText } = props;
+  const { ctaBlurb, ctaButtonText, ctaLink } = props;
   return (
     <Box className="support-cta-container" sx={{ borderTopColor: 'grey.200' }}>
       <div style={{ flex: 4, height: '100%' }}>{ctaBlurb}</div>
       <PrimaryButton
+        href={ctaLink}
```

Comment:
I was very surprised that this works, since this isn't supported on native HTML buttons. It turns out that Material UI buttons actually render a `<a>` if you pass the `href` prop (https://mui.com/material-ui/api/button/#props). Wow.